src/main/java/web/cinema/model/Movie.java

```diff
@@ -4,8 +4,10 @@
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.Table;
 
 @Entity
+@Table(name = "movies")
```

Comment:
```suggestion
@Table(name = "movie")
```
раз ти вже в усіх табл використовуєш однину, тут краще буде теж так зробити